doc/src/build/sui-local-network.md

```diff
@@ -55,9 +73,10 @@ OPTIONS:
 
 Use `sui-validator-test --help` to see these options in your console.
 
-### Making faucet requests
+## Use the local faucet
 
-To get gas coins for an address, open a new Terminal or Console window or tab. Make a cURL request and specify the address to receive the coins. Use the `sui client active-address` command to get the current active address, if needed.
+You need to have coins to pay for gas on your local network just like other networks. Use the following cURL command to get test coins from the local faucet you just installed and started. 
+To add the coins to the current active address on the local network, use the `sui client active-address` command to retrieve it. Use the `sui client addresses` command to see all of the addresses on your local network. 
```

Comment:
Can we also mention that `Alternatively you can also send coins to your Sui wallet address` and link to ## Set up a local Sui Wallet section

Reply:
Added